install.sh

```diff
@@ -2,6 +2,22 @@
 
 INSTALLER_DIR=/install
 
+# install flyway
+curl -o $INSTALLER_DIR/flyway.tgz http://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/3.2.1/flyway-commandline-3.2.1.tar.gz
+tar -C $INSTALLER_DIR/ -xzf $INSTALLER_DIR/flyway.tgz
+rm -f $INSTALLER_DIR/flyway.tgz
+mv $INSTALLER_DIR/flyway-3.2.1 /opt/flyway
+mv -f $INSTALLER_DIR/flyway.conf /opt/flyway/conf/flyway.conf
+chmod +x /opt/flyway/flyway
+ln -s /opt/flyway/flyway /usr/local/bin/flyway
+
+# install greenmail webapp to provide simple smtp service for the self-administration and administration
+curl -o $INSTALLER_DIR/greenmail.war http://central.maven.org/maven2/com/icegreen/greenmail-webapp/1.4.1/greenmail-webapp-1.4.1.war
+mv $INSTALLER_DIR/greenmail.war /var/lib/tomcat7/webapps/
```

Comment:
you can remove this step, if you download the greenmail `.war` directly into `/var/lib/tomcat7/webapps/`